cockroachcloud/pci-dss.md

```diff
@@ -0,0 +1,138 @@
+
+A comprehensive list of all actions that Cockroach Labs takes to ensure compliance with PCI DSS 3.2.1 is beyond the scope of this document. For more information, contact your Cockroach Labs account team.
+
+Compliance is a shared responsibility. Be sure to read [Responsibilities of the customer](#responsibilities-of-the-customer) to ensure that your cluster is compliant.
+
+## Responsibilities of the customer
+
+To ensure that a {{ site.data.products.dedicated }} cluster complies with PCI DSS 3.2.1, you must take additional steps, such as implementing security recommendations and carefully choosing business partners and vendors and ensuring that cardholder data is protected throughout its journey into and out of your {{ site.data.products.dedicated }} clusters.
+
+Cockroach Labs cannot provide specific advice about what is required for compliance with PCI DSS or how to implement a specific requirement. The following points help to illustrate some steps that organizations might take to help to ensure compliance.
+
+{{site.data.alerts.callout_danger}}
+The following list is provided only to illustrate a small subset of the measures you might need to take to ensure compliance.
+{{site.data.alerts.end}}
+
+- Ensure that your cluster is a {{ site.data.products.dedicated }} cluster.
+- Single Sign-On (SSO) helps you avoid storing user passwords in {{ site.data.products.db }}:
+  - [Cloud Organization Single Sign-On (SSO)](configure-cloud-org-sso.html), which allows members of your CockroachDB Cloud organization to authenticate using an identity from an identity provider (IdP) instead of using an email address and password.
+  - [Cluster SSO](cloud-sso-sql.html), which allows users to access the SQL interface of a CockroachDB cluster (whether provisioned on CockroachDB Cloud or self-hosted) with the full security of Single Sign-On (SSO), and the convenience of being able to choose from a variety of SSO identity providers, including CockroachDB Cloud, Google, Azure, GitHub, or your own self-hosted OIDC.
+- Enable [Customer-Managed Encryption Keys (CMEK)](cmek.html), which allow you to protect data at rest in a CockroachDB Dedicated cluster using a cryptographic key that is entirely within your control, hosted in a supported key-management system (KMS) platform.
+- Protect your cluster's network from the public internet by enabling the following features:
+  - [Private Clusters](private-clusters.html), which ensure that cluster nodes have no public IP addresses and that egress traffic moves over private subnets and through a highly-available NAT gateway that is unique to the cluster.
+  - [Egress Perimeter Controls](egress-perimeter-controls.html), which ensure that cluster egress operations, such as cluster backups, are restricted to a list of specified external destinations.
+- Protect data that is written from your cluster to storage on your cloud provider by enabling and using the following features:
```

Comment:
I didn't get this heading, how is it related to audit logs features?

Reply:
I thought part of the PCI-DSS aspect was to ensure that the logs are encrypted at rest in the cloud provider. Did I misunderstand? Is there a better heading that would tie together the three bullet points?